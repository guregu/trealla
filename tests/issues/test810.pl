:- initialization(main).

main :-
	catch(number_chars(N,"'-\\\n3"), Err1, writeln(err1)),
	catch(number_chars(N,"'\\\n-3"), Err1, writeln(err2)),
	writeln(done).

