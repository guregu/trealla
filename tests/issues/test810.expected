err1
err2
done
